test.py

```diff
@@ -83,10 +83,21 @@ def eval_fn(self):
             except NotImplementedError:
                 self.skipTest('Method eval is not implemented, skipping...')
 
+    def check_device_fn(self):
```

Comment:
It's not clear to me that this should be a new test, as opposed to being added as a check in `example`.

Reply:
I am not sure about this either. I thought that example tests were removed. Running `gpurun python test.py -k check_device` for all 104 check_device tests takes about 580sec (10mins). So would take less overall time if we combine this with `example` (skips many overheads).

Reply:
> So it would take less overall time if we combine this with example (skips many overheads).

Yeah, that was my thinking. Where were example tests removed? (Seems like they're still being registered in `test.py`)

Reply:
It was mentioned in this PR, maybe by accident: https://github.com/pytorch/benchmark/pull/432

Reply:
Interesting. CC @xuzhao9 to clarify, but no need to block this PR.

Reply:
> Interesting. CC @xuzhao9 to clarify, but no need to block this PR.

I actually think adding a new test here is clear that we are testing the correct target device of the models, and we may add more tests in the future to guarantee the correctness.

The name of `example` test is a bit vague, plus this is something specific to the need of JIT compiler, not all models. So I suggested we remove the `example` tests in `README.md`, and in the future we may rename it to `jit` test.

Reply:
Ah, I see. Thanks!